misc/destinations/README.md

```diff
@@ -67,7 +67,7 @@ Authentication=NoAuthentication
 HTML5.Timeout=60000
 WebIDEEnabled=true
 ProxyType=Internet
-URL=https\://services.odata.org
+URL=https://services.odata.org
```

Comment:
You need to revert this, this is there soo customers can import the properties into SAP BTP